src/app/pages/review/review.component.html

```diff
@@ -0,0 +1,57 @@
+<div>
+  <h1 nz-typography>Review images</h1>
+</div>
+<nz-empty *ngIf="!listRegions.length" [nzNotFoundContent]="emptyContent">
+  <ng-template #emptyContent>
+    <p nz-typography>No text region needs verifying</p>
+  </ng-template>
+</nz-empty>
+<br>
+<div *ngIf="listRegions.length" nz-row [nzGutter]="{ xs: 8, sm: 16, md: 24, lg: 32, xl: 32, xxl: 32 }">
+  <div
+  *ngFor="let item of listRegions"
+  nz-col
+  nzSpan="24"
+  nzLg="16"
+  nzXl="12"
+  style="margin-top: 8px; margin-bottom: 8px"
+  > 
+  <nz-card [nzHoverable]="true" style="width:480px"  [nzCover]="coverTemplate" [nzActions]="[submitLabel]">
+  </nz-card>
+  <ng-template #coverTemplate>
+      <app-zoomable-image
+        [imageSrc]="item.imageUrl"
+        [region]="item.region.region.vertices"
+      ></app-zoomable-image>
+  </ng-template>
+  <ng-template #submitLabel>
+  <div>
+    <input
+      id="label-input"
```

Comment:
Hạn chế sử dụng `id`, trừ khi ông biết chắc chắn là chỉ có 1 object loại này trên toàn bộ trang web.
`id` hướng tới việc định danh HTML element độc lập, phân biệt với nhau - khác với `class` là các object cùng loại.
Angular sẽ đổi tên class cho mình khi dịch chương trình, nhưng không đổi tên id. Do đó nếu lỡ có 2 element có cùng id (ví dụ cái `label-input` này và cái `label-input` ở trang Label) thì style của chúng sẽ bị áp dụng cho thằng còn lại -> rất không tốt.